tests/conftest.py

```diff
@@ -153,7 +153,7 @@ def inner(
         http_client: Union[SyncHTTPClient, AsyncHTTPClient],
         data_list: list,
         status_code: int = 200,
-        headers: Mapping[str, str] = None,
+        headers: Optional[Mapping[str, str]] = None,
```

Comment:
Maybe we should point mypy at more of our files?

Reply:
Seemingly we should be able to open it up to all `workos/` files. That's how I was running it locally. If we're having a mypy mismatch we should adjust settings to make sure it runs how we'd like in CI.